src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartTokenFormatterFormatRangeTests.cs

```diff
@@ -32,10 +32,11 @@
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Formatting.Indentation
 {
     [UseExportProvider]
+    [Trait(Traits.Feature, Traits.Features.Formatting)]
```

Comment:
💭 Maybe just `SmartTokenFormatting` was meant here?

Reply:
honestly, i would remove the disparate traits here and just go with one.  When i'm genuinely testing formatting, i always want to run all of these. :)